deploy/helm/sumologic/README.md

```diff
@@ -482,6 +479,8 @@ The following table lists the configurable parameters of the Sumo Logic chart an
 | `pvcCleaner.job.podAnnotations`                                                       | Additional annotations for for the pvcCleaner container.                                                                                                                                                                                                                                                   | `{}`                                                                                                                                                                                 |
 | `pvcCleaner.job.schedule`                                                             | Schedule for cronJobs                                                                                                                                                                                                                                                                                      | `*/15 * * * *`                                                                                                                                                                       |
 | `pvcCleaner.job.securityContext`                                                      | The securityContext configuration for the pvcCleaner.                                                                                                                                                                                                                                                      | `{"runAsUser": 1000}`                                                                                                                                                                |
+| `debug.instrumentation.tracesSampler.print`                                           | Set to `true` to print logs processed by log collector on standard output.                                                                                                                                                                                                                                 | `false`                                                                                                                                                                              |
+| `debug.instrumentation.tracesSampler.stopLogsIngestion`                               | Set to `true` to prevent sending log collector logs to Sumo Logic. It prevents increasing ingestion and costs in case of huge amount of log collector logs.                                                                                                                                                | `false`                                                                                                                                                                              |
```

Comment:
```suggestion
| `debug.instrumentation.tracesSampler.stopLogsIngestion`                               | Set to `true` to prevent sending log collector logs to Sumo Logic. It prevents increasing ingestion and costs in case of huge amount of traces-sampler logs.                                                                                                                                                | `false`                                                                                                                                                                              |
```